api/XDS_PROTOCOL.md

```diff
@@ -286,50 +286,50 @@ admin:
 
 ```
 
-### Incremental xDS
+### Delta xDS
 
-Incremental xDS is a separate xDS endpoint available for ADS, CDS and RDS that
+Delta xDS is a separate xDS endpoint available for ADS, CDS and RDS that
 allows:
 
-  * Incremental updates of the list of tracked resources by the xDS client.
+  * Delta updates of the list of tracked resources by the xDS client.
```

Comment:
Ah, I think we still need to clarify terminology here a bit more cleanly. After today's call, I was left with the impression that we had:
* Delta xDS referring only to the fact that when the server sends to client, it sends + or - resources.
* On-demand xDS referring to lazy fetch from client of resources from server.
* Incremental xDS (?) referring to the combo of the two.

In places where we have common ground between delta + on-demand, I think it's fine to keep the incremental naming that we have today, for example we can keep `IncrementalDiscoveryRequest`.

Reply:
Similar to above. It is just delta; it supports on-demand in the sense of making it not prohibitively expensive.